src/ctr_exit.c

```diff
@@ -215,6 +215,16 @@ void reap_children()
 		;
 }
 
+void cleanup_socket_dir_symlink()
+{
+	/* A symbolic link might be created at {opt_socket_path}/{opt_cuuid} if the container
+	   is created successfully.
+	   This function will take care of removing the link when the conmon process exits. */
+	char *base_path = g_build_filename(opt_socket_path, opt_cuuid, NULL);
```

Comment:
Happened to see this go by in my notifications;
https://github.com/containers/conmon/pull/387

Reply:
I didn't do that as we're going to exit, but good point about the static analyzers 👍